euth/flashpoll/templates/euth_flashpoll/includes/project_form_result_flashpoll_tab.html

```diff
@@ -33,7 +39,7 @@ <h3 class="form-section-heading">{% trans 'Result '%}</h3>
 					</div>
 					{% endif %}
 				{% endfor %}
-			{% endif %}
+			{% endif %}            
```

Comment:
You added some [trailing white space](https://softwareengineering.stackexchange.com/questions/121555/why-is-trailing-whitespace-a-big-deal) to this line, please don't do that.